blobfuse/fileapis.cpp

```diff
@@ -217,7 +214,8 @@ int azs_create(const char *path, mode_t mode, struct fuse_file_info *fi)
     ensure_files_directory_exists_in_cache(mntPathString);
 
     // FUSE will set the O_CREAT and O_WRONLY flags, but not O_EXCL, which is generally assumed for 'create' semantics.
-    res = open(mntPath, fi->flags | O_EXCL, mode);
+    // overwrite mode_t because user should always 770 permission until we have full ACL support
+    res = open(mntPath, fi->flags | O_EXCL, 0770);
```

Comment:
maybe 0777? we always mount with -o allow_other

Reply:
I just tested it - these modes does not really matter. Even when it is 770, some other user is able to read, write and execute if you mount it with allow_other option. Only way these permissions get enforced is to mount with -o default_permissions option. Then the kernel starts enforcing these, and you can no longer access the files from other users.

I'll leave these as is until we implement ACL support.

Reply:
The issue is that user will see a permission which doesn't reflect reality.  Exactly as you told in your comment.